src/Transports.Subscriptions.Abstractions/SubscriptionServer.cs

```diff
@@ -136,5 +137,7 @@ private void LinkToTransportWriter()
             TransportWriter = Transport.Writer;
             _logger.LogDebug("Writer pipeline created");
         }
+
+        public void Dispose() => (Subscriptions as IDisposable)?.Dispose();
```

Comment:
It's not a "DI-way", but OK for now.

Reply:
This is calling SubscriptionManager.Dispose if I remember right

Reply:
Confirmed.